.github/actions/e2e_test/action.yml

```diff
@@ -36,13 +39,23 @@ inputs:
   azureResourceGroup:
     description: "The resource group to use"
     required: false
-  sonobuoyTestSuiteCmd:
-    description: "Which tests should be run? Check README for guidance!"
+  test:
+    description: "The test to run. Can currently be on of [sonobuoy, autoscaling]."
```

Comment:
```suggestion
    description: "The test to run. Can currently be one of [sonobuoy, autoscaling]."
```

Reply:
Ahhrg, there are strings missing in the list, too.